### ISSUE TITLE
Help Web not working on macOS

### ISSUE THREAD

**Issue by VitGottwald** [zowe/zowe-cli#738]
When I run `zowe --help-web` on my mac, I get the following output with an error:

```
tmp zowe --help-web
You are running in an environment with no graphical interface.
Alternatively, you can run 'zowe --help' for text-based help.

If you are running in an X Window environment,
ensure that your DISPLAY environment variable is set.
For example, type the following:
    echo $DISPLAY
If it is not set, assign a valid value. For example:
    export DISPLAY=:0.0
Then try the --help-web option again.
```

I would expect zowe cli to run [open](https://ss64.com/osx/open.html) on an html file so that my default browser would show the help.

**Comment by tjohnsonBCM** [comment]
Related to https://github.com/zowe/imperative/issues/322

**Comment by tjohnsonBCM** [comment]
This should be fixed in v6.17.3 or newer of `@zowe/cli`. Feel free to reopen if web help still fails to load on macOS.
